src/components/Button/PlayButton.js

```diff
@@ -0,0 +1,15 @@
+import React, { PropTypes } from 'react';
+
+const commonPlayBtnClasses = 'fa fa-play w3 h3 f3 absolute z-1 gray items-center justify-center br-pill pointer card-play-btn'
```

Comment:
why the `card-play-btn` here? I thought we were sticking to tachyons class names?